pdk/1.0/modules/ROOT/pages/policies-pdk-apply-policies.adoc

```diff
@@ -12,19 +12,82 @@ To learn more about defining configuration parameters, see xref:policies-pdk-cre
 
 == Before You Begin
 
-* xref:policies-pdk-publish-policies.adoc[Upload your Custom Policy to Exchange]
+* xref:policies-pdk-compile-policies.adoc[Compile your custom policy].
+* If applying your policy to Flex Gateway running in Connected Mode, xref:policies-pdk-publish-policies.adoc[upload your custom policy to exchange].
 
-== Connected Mode
+== Apply Policies in Connected Mode
 
 When you apply a custom policy in Connected Mode, the configuration parameters and metadata defined in `gcl.yaml` appear in the API Manager UI.
 
 To apply a custom policy in Connected Mode, see xref:gateway::policies-included-apply.adoc[Apply a Policy in Connected Mode]. 
 
-== Local Mode
+=== Update Policies in Connected Mode
+
+To update a custom policy in API Manager:
+
+. Go to *Anypoint Platform > API Manager*.
+. In *API Administration* click the name of the API to which to apply a policy.
+. From the left navigation menu, click *Policies*.
+. Click the more options button (image:more-options-menu.png[1%,1%]) of the policy you want to update, and then click *Edit configuration*.
+. Select the *latest* version for *Policy version* configuration parameter.
+. Click *Save*.
+. Click the more options button (image:more-options-menu.png[1%,1%]) of the policy you want to update, and then click *Check for implementation updates*.
++
+For more information about implementation updates, see xref:exchange::manage-versions.adoc#update-policy-implementations[Update Policy Implementations].
+. In the confirmation dialog box, click *Update all*.
+
+
+
+[[apply-policies-in-local-mode]]
+== Apply Policies in Local Mode
 
 For Flex Gateway running in Local Mode, you apply custom policies using xref:gateway::flex-local-configuration-reference-guide.adoc[local declarative configuration files]. You specify the location of these configuration files when you first run Flex Gateway.
 
-To apply a custom policy in Local Mode, use either xref:gateway::flex-local-secure-api-with-basic-auth-policy.adoc[] or xref:gateway::flex-local-secure-api-with-auto-policy.adoc[] as an example. Replace the example configuration with your policy's configuration parameters.
+To create the local declarative files:
+
+. In the `target/wasm32-wasi/release` PDK directory, copy both of the following files:
++
+* `<your_custom_policy>_definition.yaml`
+* `<your_custom_policy>_implementation.yaml`
++
+. Paste the two files in the `/etc/mulesoft/flex-gateway/conf.d/` Flex Gateway configuration directory.
++
+. Create a policy binding configuration file with a `.yaml` file extension to bind the policy to an API instance:
++
+* Give the file a custom name.
+* Save the file in the Flex Gateway configuration directory `/etc/mulesoft/flex-gateway/conf.d/custom`. This directory can contain multiple configuration files.
```

Comment:
I'm fairly certain this refers to the config directory in production environment, but I'm not 100% sure. Does it need to be made explicit?